### ISSUE TITLE
RStudio window becomes blank after some time when training a Tensorflow model on Mac

### ISSUE THREAD

**Issue by anirban-mukherjee** [rstudio/rstudio#7260]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.1054
    OS Version      : Mac OS X 10.15.5
    R Version       : 4.0.2

### Steps to reproduce the problem

Train a model in Tensorflow. RStudio window becomes blank after some time (see screenshot below).

<img width="1527" alt="Screenshot 2020-07-02 at 9 34 16 AM" src="https://user-images.githubusercontent.com/12909536/86306584-38363d00-bc47-11ea-96e1-f8a94e1f3222.png">

### Describe the problem in detail

In both stable and preview releases of RStudio desktop for Mac, the RStudio window becomes blank after some time when R is running Keras/Tensorflow/Reticulate code (through the R library). I don't know if this is a Reticulate/Python issue or a Keras/Tensorflow issue. There is no problem when I am not running Keras/Tensorflow/Reticulate code. It's only when I am training a large Tensorflow model that the window becomes blank.

This has been going on for a while (more than a month) but I have not had the time to report it. I have seen it in many different versions of RStudio. I have cleared RStudio's state by deleting ~/.rstudio-desktop and re-installed RStudio multiple times. I have also used both software and OpenGL rendering. None of that makes any difference. Furthermore, if I purl the R Markdown code to R script and then run it on Terminal, then the code executes fine and RStudio behaves as expected. It's only when I source from R Markdown to console that I observe this issue.

The toolbar does not work if the window is blank. I can, however, launch a new window, in which case everything works in the new window but the old new window remains blank. Even after the Tensorflow code finishes, the window remains blank. R and Tensorflow continue computing as expected (as reported through activity manager).

The RStudio window just turned blank. I have attached the diagnostics report generated through Terminal.

[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/4861502/diagnostics-report.txt)

### Describe the behavior you expected

RStudio window should not become blank.

- [ X] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [ X] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ X] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [ X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by anirban-mukherjee** [comment]
FWIW, if I purl the R Markdown code to a R script and run it on the Terminal, then the code executes fine and RStudio behaves as expected. It's only when I source to console that this issue occurs.

**Comment by R-Hannibal** [comment]
@anirban-mukherjee - Are you still experiencing this issue? If so, could you provide more detail/an example on the TensorFlow model you are training as well as your environment?

We have  not been able to reproduce this issue in house.

**Comment by anirban-mukherjee** [comment]
Unfortunately, I am not sure which model I was training and I have also not been used TensorFlow from R in recent months. I *think* this occured when I used an early stopping callback and had verbose "on" (I don't remember if I used 1 or 2). I think my fix was to set verbose to 0 and then I didn't see this issue. I also didn't see this issue if I did not use the early stopping callback. If you cannot replicate then I suggest we close the bug report as  I don't have further suggestions or inputs, sorry.
